utils.go

```diff
@@ -322,3 +322,73 @@ func initWalletLoader(chainParams *chaincfg.Params, walletDataDir, walletDbDrive
 
 	return walletLoader
 }
+
+func s(x float64) string {
+	if int(x) == 1 {
+		return ""
+	}
+	return "s"
+}
+
+func TimeElapsed(now time.Time, then time.Time, full bool) string {
```

Comment:
I should be able to decide the format i want the time to be example `m`, `min(s)`, `minute(s)`

Reply:
So this is a Dcrlibwallet method used on godcr with timestamp. https://github.com/planetdecred/godcr/blob/d9f384c76ab1068b3f01b90810207dcd1c104244/wallet/commands.go#L117

This and many more methods are in the dcrlibwallet utils.go file.

Having this, it will help eliminate repeating same method on godcr and the other platform.
especially this as the logic is the same.
https://github.com/planetdecred/godcr/blob/d9f384c76ab1068b3f01b90810207dcd1c104244/ui/page/components/components.go#L439

Reply:
oh okay, noted, thanks for clearing it up 👍🏼